{{cookiecutter.project_slug}}/MANIFEST.in

```diff
@@ -21,6 +23,9 @@ exclude .editorconfig
 exclude .pre-commit-config.yaml
 exclude .readthedocs.yml
 exclude .yamllint.yaml
+{% if cookiecutter.open_source_license != 'Not open source' -%}
+include .zenodo.json
```

Comment:
An include among all the excludes?  Should be group all includes and excludes together?

Reply:
Good catch!